sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -844,6 +837,7 @@ export class ManagementClient extends LinkEntity {
         const decodedMessage = RheaMessageUtil.decode(msg.message);
         const message = new ServiceBusMessageImpl(
           this._context,
+          this.entityPath,
```

Comment:
I really like the fallout from this change.

Even though this "looks" like more work it's more clearly documenting what is depended on. Loving it.